style-guide: add emphasis section (#11971)
tldr-pages/tldr: contributing-guides/style-guide.md

 `ls`
```

## Emphasis

> - Don't use *itallics* and **bold** in the pages. These are reserved for client emphasis of placeholders.
> - Don't uppercase words unnecessarily. These are reserved to special cases like acronyms, brand names and notes on a description.

Reviewer: ```suggestion
- Do not use *italics* and **boldface** in the pages. These are reserved for client emphasis of placeholders.
- Do not uppercase words unnecessarily. These are reserved to special cases like acronyms, brand names and notes on a description.
```

> - Do not uppercase words unnecessarily. These are reserved to special cases like acronyms, brand names and notes on a description.

Reviewer: ```suggestion
- Do not use ALL CAPS unnecessarily. These are reserved for special cases like acronyms, brand names and notes on a description.
```

Reviewer 2: Maybe we can rephrase it to something like "do not capitalize words in middle of sentences unnecessarily ...". I don't think we use all caps in non-acronym instances in pages.

Reviewer: > Maybe we can rephrase it to something like "do not capitalize words in middle of sentences unnecessarily ...". I don't think we use all caps in non-acronym instances in pages.

Are we talking about instances of words written in ALL CAPS or words that are Unecessarily Capitalised Only At The First Character?

Reviewer 2: > Are we talking about instances of words written in ALL CAPS or words that are Unecessarily Capitalized Only At The First Character? This should be clarified in the proposed change.

Exactly, we are talking about the latter (words that are unnecessarily capitalized) here.

```suggestion
- Do not capitalize words in descriptions unnecessarily. We allow capitalization of starting words and special cases like acronyms, brand names and notes.
```

Author: > Exactly, we are talking about the latter (words that are unnecessarily capitalized) here.

Well, I was talking about words writer in ALL CAPS :sweat_smile:, I thought it's more common for people to do this for emphasis. 

Since we had thought of different things, I think it would be nice to document both of them.

Reviewer: > > Are we talking about instances of words written in ALL CAPS or words that are Unecessarily Capitalized Only At The First Character? This should be clarified in the proposed change.
> 
> Exactly, we are talking about the latter (words that are unnecessarily capitalized) here.

I don't think that this should be covered in the style guide since it is already covered by the usual English grammar rules.

Reviewer 2: I agree with @gutjuri here, we could drop this point and we can change the above point (with -) to a normal paragraph.
